[ELEC2811] Exam janvier 2019 (#769)
Gp2mv3/Syntheses: src/q7/instrusensors-ELEC2811/exam/2019/Janvier/All/instrusensors-ELEC2811-exam-2019-Janvier-All.tex

  \end{itemize}
  A direct sensor must convert directly a non electrical stimulus into electrical signal or modify an electrical signal.
\end{solution}

\section{}

As regards a piezoresistance with a strain gauge, select the \textbf{false} proposition:

\begin{itemize}
    \item The strain gauge factor of the silicon highly depends on its doping.
    \item The strain gauge factor for metals can vary over large values between $-120$ and 200.
    \item The strain gauge factor depends on the Poisson's coefficients and the relative variation of the resistivity with the stress.
    \item The strain gauge factor measures the ratio between the relative variation of the resistance and the strain.
    \item 
\end{itemize}

\begin{solution}
\begin{itemize}
  \item True (polysilicon -30 -- 30, single crystal silicon -125 -- 200)
  \item False (Metals 2 -- 5, single crystal silicon -125 -- 200) 
  \item True
    $$S_e = 1 + 2 \nu + \frac{\Delta\rho/\rho}{\epsilon}$$
  \item True
    $$S_e = \frac{\Delta R/R}{\epsilon}$$
\end{itemize}
\end{solution}
\section{}

Give four oscillators and their type (sine or binary).

\begin{solution}
  \begin{itemize}
    \item NE555 oscillator: binary

>     \item flip flop oscillator: bniary

Reviewer: ```suggestion
    \item flip flop oscillator: binary
```

    \item Schmitt trigger oscillator: binary
    \item quartz oscillators: sine
    \item Collpitts oscillator: sine
  \end{itemize}
\end{solution}

\section{}

Give three applications based on the piezoelectric effect.

\begin{solution}
  \begin{itemize}
    \item Quartz oscillator
    \item Contact microphones

>     \item Stain gauges

Reviewer: ```suggestion
    \item Strain gauges
```

    \includegraphics[width=0.6\textwidth]{quest11.png}
\end{figure}

\begin{solution}

> 	The double arrow represent the dynamic range fo the sensor

Reviewer: ```suggestion
	The double arrow represents the dynamic range for the sensor
```

    \includegraphics[scale=0.45]{quest13.png}
\end{figure}

\begin{solution}

>   We use the exposure time as a degree of freedom. We take multiple images with different exposure times to have a good resolution for each pixel

Reviewer: ```suggestion
  We use the exposure time as a degree of freedom. We take multiple images with different exposure times to have a good resolution for each pixel.
```

    \item Give a sensor that could be used to measure this distance.
    
\end{enumerate}

\begin{solution}
  \begin{itemize}
    \item We want $|V_{TB} - V_{J}| > 0.4\ [mV]$ to have margin for the error

      $$R_J - R_{TB} > \frac{0.4}{0.1} = 4\ [\Omega]$$

>       i$$R_0(\alpha_{TB} - \alpha_J)\left(\frac{h_{\text{liquid}}}{h_{\text{glass}}}\right)^2 > 4$$

Reviewer: Normal ce "i" ?
```suggestion
      $$R_0(\alpha_{TB} - \alpha_J)\left(\frac{h_{\text{liquid}}}{h_{\text{glass}}}\right)^2 > 4$$
```
